### ISSUE TITLE
Add Stress Test

### ISSUE THREAD

**Issue by ghost** [vorticalbox/Pymark#4]
I'd like to suggest the addition of a stress test which would effectively be Pymark running the multi threaded Float or Integer calculations infinitely until the user wishes for it the stress test to stop.

This would be useful for those wanting to test overclocks and general system stability.

**Comment by vorticalbox** [comment]
I agree I am slowly looking back at my old projects, only problem using this method it will only test int/float.

As far as I know to truly bring a CPU to its knees you would want to hit up the instruction set as well.
